x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/RollupShardIndexer.java

```diff
@@ -352,7 +361,6 @@ public void postCollection() throws IOException {
                 XContentBuilder doc = rollupBucketBuilder.buildRollupDocument();
                 indexBucket(doc);
             }
-            bulkProcessor.flush();
```

Comment:
Not needed because this would happen during closing of `BulkProcessor2`?

Reply:
That's right. There's no need to explicitly flush BulkProcessor2 (it doesn't even have a public flush method).